packages/pages/src/components/image/types.ts

```diff
@@ -70,6 +70,25 @@ export const ImageLayoutOption = {
 export type ImageLayout =
   typeof ImageLayoutOption[keyof typeof ImageLayoutOption];
 
+/**
+ * Loading option on the Image component
+ *
+ * @public
+ */
+export const LoadingOption = {
+  /** The default loading state, will wait for image to enter viewport before loading */
+  LAZY: "lazy",
+  /** Force image to load immediately */
+  EAGER: "eager",
+} as const;
+
+/**
+ * The type definition for image loading.
+ *
+ * @public
+ */
+export type Loading = typeof LoadingOption[keyof typeof LoadingOption];
```

Comment:
I don't think we're using the LoadingOption const for anything except creating this type, so it could probably be simplified to just `loading?: "eager" | "lazy"` down below and we wouldn't need the new type

Reply:
that would match
![image](https://user-images.githubusercontent.com/23005393/199535561-407ec6a9-9146-475b-a2a6-b9e215c0d6e4.png)

Reply:
Yeah I'll switch it back to this